spec/models/report_spec.rb

```diff
@@ -31,4 +41,78 @@
     report = build(:report, deadline: Date.yesterday)
     expect(report).not_to be_valid
   end
+
+  context "when in the first quarter of the financial year - April to March" do
+    it "sets the financial quarter to 1" do
+      ["1 April 2020", "30 June 2020"].each do |date|
+        travel_to(Date.parse(date)) do
+          report = Report.new
+          expect(report.financial_quarter).to eql 1
+        end
+      end
+    end
+  end
+
+  context "when in the second quarter of the financial year - May to July" do
+    it "sets the financial quarter to 2" do
+      ["1 July 2020", "30 September 2020"].each do |date|
+        travel_to(Date.parse(date)) do
+          report = Report.new
+          expect(report.financial_quarter).to eql 2
+        end
+      end
+    end
+  end
+
+  context "when in the third quarter of the financial year - October to December" do
+    it "sets the financial quarter to 3" do
+      ["1 October 2020", "31 December 2020"].each do |date|
+        travel_to(Date.parse(date)) do
+          report = Report.new
+          expect(report.financial_quarter).to eql 3
+        end
+      end
+    end
+  end
+
+  context "when in the fourth quarter of the financial year - January to March" do
+    it "sets the financial quarter to 4" do
+      ["1 January 2020", "31 March 2020"].each do |date|
+        travel_to(Date.parse(date)) do
+          report = Report.new
+          expect(report.financial_quarter).to eql 4
+        end
+      end
+    end
+  end
+
+  context "when in the 2020 - 2021 financial year" do
```

Comment:
... and same here `describe '#financial_year' do`.

Not sure this `context` is needed at all, because it's describing what the test is *doing* rather than what it is *testing*.

Reply:
I take your point - I do want to make it clear to devlopers who may not know about financial years spanning two calendars years - I've removed the context and add the financial year to the inner context - let me know what you think?